src/components/Header/Header.styl

```diff
@@ -0,0 +1,18 @@
+@require 'settings/breakpoints'
+
+.HeaderColor_default
+    background-color var(--white)
+
+.HeaderColor_primary
+    background-color var(--primary)
+
+.Header_fixed
+    z-index 1
+    position fixed
+    top 3rem
+    right 0
+    left 221px
```

Comment:
Where does this value come from ?

Reply:
I remove this part because it's not necessary on this PR 👍